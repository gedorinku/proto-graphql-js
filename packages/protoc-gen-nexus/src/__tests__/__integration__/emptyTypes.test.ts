import { testSchemaGeneration } from "../__helpers__/process.test.helper";

testSchemaGeneration("empty_types", "native protobuf");
testSchemaGeneration("empty_types", "protobufjs");
