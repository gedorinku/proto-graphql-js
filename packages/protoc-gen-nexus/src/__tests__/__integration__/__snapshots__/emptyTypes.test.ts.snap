// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`schema generation with native protobuf can make GraphQL schema from generated DSLs: schema.graphql 1`] = `
"### This file was generated by Nexus Schema
### Do not make changes to this file directly


type EmptyMessage {
  \\"\\"\\"noop field\\"\\"\\"
  _: Boolean
}

input EmptyMessageInput {
  \\"\\"\\"noop field\\"\\"\\"
  _: Boolean
}

type Query {
  ok: Boolean!
}
"
`;

exports[`schema generation with protobufjs can make GraphQL schema from generated DSLs: schema.graphql 1`] = `
"### This file was generated by Nexus Schema
### Do not make changes to this file directly


type EmptyMessage {
  \\"\\"\\"noop field\\"\\"\\"
  _: Boolean
}

input EmptyMessageInput {
  \\"\\"\\"noop field\\"\\"\\"
  _: Boolean
}

type Query {
  ok: Boolean!
}
"
`;
